src/Components/PairingForm.js

```diff
@@ -0,0 +1,61 @@
+import React from 'react';
+import { useDispatch, useSelector } from 'react-redux';
+import { findGame } from '../redux/actions/boardGame';
+import { setLiquorType, setRandomDrink } from '../redux/actions/liquorSearch'
+
+const PairingForm = () => {
+
+    const dispatch = useDispatch()
+
+    const games = useSelector(state => state.boardGame.allGamesData.games)
+    const type = useSelector(state => state.liquorSearch.liquorSearchWord)
+
+    const gameNames = games.map((game) => {
+        return (
+            <option key={game.id} value={game.name} />
+        )
+    })
+
+    const findRandomDrink = (event) => {
+        event.preventDefault()
+        fetch(`https://www.thecocktaildb.com/api/json/v1/1/filter.php?i=${type}`)
```

Comment:
quick housekeeping note here - we should make moves to separate all API calls into a separate API calls file.

Reply:
Same thought!